packages/matchbox/src/components/Tooltip/Tooltip.js

```diff
@@ -89,11 +100,16 @@ function Tooltip(props) {
   }
 
   function renderActivator({ activatorRef }) {
+    const { as = 'span' } = props;
+
     return (
       <Box
-        display="inline-block"
-        onMouseOver={handleMouseOver}
-        onMouseOut={handleMouseOut}
+        as={as}
+        display={as === 'span' ? 'inline-block' : null}
```

Comment:
Previous implementation forced inline-block and old tooltips never worked correctly on block level elements, this shouldn't be a breaking change

Reply:
This comment should probably be in code because "as" is not restricted to only block elements and I could set "as" to "b" and it won't behave the same.